### PR TITLE
Remove quotes from singleuser.sh

### PR DESCRIPTION
**Pull request by dmpe**
Being consistent with other args

Thanks goes to my colleague at work who saw it.

### CODE REVIEW

**Comment by dmpe** [comment]
Haha, it is not so simple. The directory may contain a whitespace and therefore it was implemented. 

In principle, arrays would be better.

**Comment by parente** [comment]
Thanks for submitting this PR. Is there an issue the quoting is causing when specifying a the notebook directory?

**Comment by dmpe** [comment]
Not necessary that I have come across, but i had to build a custom docker image and was "scanning" files. 
Quotes within quotes can be quite confusing and different shells (OS/versions) may treat them differently as well. Maybe it is a question of style too, i am more used to apply in these cases bash arrays.

File: base-notebook/start-singleuser.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ fi
 # These won't be passed from DockerSpawner 0.9,
 # so avoid specifying --arg=empty-string
 if [ ! -z "$NOTEBOOK_DIR" ]; then
-  NOTEBOOK_ARGS="--notebook-dir='$NOTEBOOK_DIR' $NOTEBOOK_ARGS"
+  NOTEBOOK_ARGS="--notebook-dir=$NOTEBOOK_DIR $NOTEBOOK_ARGS"
```
Comment (consideRatio) [inline base-notebook/start-singleuser.sh:17]:
```suggestion
  NOTEBOOK_ARGS="--notebook-dir=\"$NOTEBOOK_DIR\" $NOTEBOOK_ARGS"
```

**Comment by consideRatio** [comment]
The reason that the quote exists is to not crash if the file path contains spaces.

Using `'` quotes instead of `"` quotes (that could be introduced with `\"` will make paths like `~/work` fail to work though, as that require bash to expand them to `/home/jovyan/work`.

I think if it could make sense to quote like this instead `\"` instead though to allow NOTEBOOK_DIR='~/work' to be expanded to the user fold or similar.

Oh wait... This is all about something that is deprecated and I would advocate we remove all together as it was deprecated 3 years ago. I've added a commit to remove this section in #1052 which ended up being a major refactoring of start.sh / start-notebook.sh / start-singleuser.sh.

I suggest we close this PR, @dmpe what do you think?

**Pull request closed without merging**